### ISSUE TITLE
Saving meta boxes accessibility

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#4188]
Just seen https://github.com/WordPress/gutenberg/pull/4170 and wondering what should happen when saving metaboxes. While )I guess?) there's a visible feedback, there should be an audible feedback too.
Without that, blind users or low vision users using a screen reader would hear just a resounding silence and no feedback at all.

**Comment by youknowriad** [comment]
I'm not sure the users should know about the meta boxes being saved at all. For users, there's no difference between saving a post and saving a meta box. So, we should make sure there's feedback for saving posts and that should be enough.

**Comment by karmatosed** [comment]
@afercia is there audible feedback right now using something like ACF?

**Comment by afercia** [comment]
I did not test this in depth, sorry. Now that there are audible messages when saving the post ,I totally agree with @youknowriad that is enough!

@karmatosed: outside of Gutenberg, the feedback for users is the page reload 🙂 screen readers announce a page reload, they start reading the page from the beginning so that's what makes users understand something happened after they submit a form.
Instead, in a world where page reloads are banned, there's no feedback at all. Users click a button that says "Publish" or "Update" and after that there's just a resounding silence. Thus, there's the need to provide some alternative feedback to replace the one given by a page reload.
